app/main/lib/presto.py

```diff
@@ -14,14 +15,18 @@ def get_similar_items_callback_url(alegre_host, similarity_type):
         return f"{alegre_host}/presto/receive/search_item/{similarity_type}"
 
     @staticmethod
-    def send_request(presto_host, model_key, callback_url, message):
-        return requests.post(f"{presto_host}/process_item/{model_key}", json={
+    def send_request(presto_host, model_key, callback_url, message, requires_callback=True):
+        data = {
             "callback_url": callback_url,
```

Comment:
oh, I see the callback url is in the data payload?

Reply:
Yeah that is what I'm imagining - if we explicitly pass it in, that'll make it much more flexible to other use cases rather than just obliquely implying them with some other structure?